tools/lint-md.js

```diff
@@ -4482,155 +4482,155 @@ var escapeStringRegexp = function (str) {
 	return str.replace(matchOperatorsRe, '\\$&');
 };
 
```

Comment:
Is the new executable bit intentional?

Reply:
Not sure how that happened, but I've fixed it. Thanks!